### PR TITLE
config: Integrate Prettier for code formatting

### LINKED ISSUE #148: Set Up Prettier
- [ ] Set up prettier
- [ ] Set up a GH action to run prettier on push

### PR DESCRIPTION
**Pull request by rohan-b-84**
Closes #148 
- [x] Set up prettier
- [x] Set up a GH action to run prettier on push

### CODE REVIEW

**Comment by rajivharlalka** [comment]
@rohan-b-84  update?

**Comment by rohan-b-84** [comment]
I'm traveling, will write GH action by tonight

**Comment by rajivharlalka** [comment]
Ah, okay cool cool

**Review by harshkhandeparkar — DISMISSED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+      - run: pnpm install
+      - run: pnpm format
```
Comment (harshkhandeparkar) [inline .github/workflows/main.yml:18]: Is the formatted code pushed?

**Comment by harshkhandeparkar** [comment]
Let's the `prettier-setup` branch to the list to see if it works. We can remove it after testing. @rohan-b-84

**Review by harshkhandeparkar — DISMISSED** [review]

**Review by harshkhandeparkar — APPROVED** [review]

**Review by harshkhandeparkar — APPROVED** [review]
Thanks for this pretty PR!

**Pull request merged**